### ISSUE TITLE
Unlock the screen before starting gapir

### ISSUE THREAD

**Issue by pmuetschard** [google/gapid#2809]
Use the same tricks for unlocking the screen for tracing before starting gapir.

**Comment by hevrard** [comment]
Duplicate of #2791 

I would like to keep this one as an opportunity for first-time contribution.

**Comment by hevrard** [comment]
Closing as duplicate
